Cascade user destroy to its attributes

rafamanzo opened the pull request:
Otherwise on databases that implement foreign keys it will raise a exception
on destroying. See:

https://lists.ime.usp.br/archives/mezuro-core/2015-August/005991.html

danielkza commented:
I wonder why this never showed up in our environments before. It's also the second integrity-related issue this week, mezuro/kalibro_processor#133 would have been caught with better DB constraints.

rafamanzo committed:
Cascade user destroy to its attributes

Otherwise on databases that implement foreign keys it will raise a exception
on destroying. See:

https://lists.ime.usp.br/archives/mezuro-core/2015-August/005991.html

rafamanzo commented:
In this case, we've never cought this before because we develop and test (incliding Travis) using SQLite which does not support foreign keys. But, we use postgres under production.

I think we should run the Travis build using postgres.
